packages/react-core/src/components/NotificationBadge/NotificationBadge.tsx

```diff
@@ -16,13 +20,21 @@ export interface NotificationBadgeProps extends ButtonProps {
 
 export const NotificationBadge: React.FunctionComponent<NotificationBadgeProps> = ({
   isRead = false,
+  isNeedingAttention = false,
+  count = 0,
   className,
   children,
   ...props
 }: NotificationBadgeProps) => (
   <Button variant={ButtonVariant.plain} className={className} {...props}>
-    <span className={css(styles.notificationBadge, isRead ? styles.modifiers.read : styles.modifiers.unread)}>
+    <span
+      className={css(
+        styles.notificationBadge,
+        isRead ? styles.modifiers.read : isNeedingAttention ? styles.modifiers.attention : styles.modifiers.unread
```

Comment:
Isn't this a variant? Based on the [core examples](https://pf4.patternfly.org/documentation/core/components/notificationbadge) I assume a badge CANNOT be "isRead" and "isNeedingAttention."

If that is the case I think it'd be good to add a variant prop rather than `isNeedingAttention`. To not be a breaking change we can have `isRead` just set the variant to `read` or `unread` if not defined.